airflow/cli/commands/webserver_command.py

```diff
@@ -258,15 +258,15 @@ def _check_workers(self) -> None:
             num_workers_running = self._get_num_workers_running()
             if num_workers_running < self.num_workers_expected:
                 new_worker_count = min(
-                    num_workers_running - self.worker_refresh_batch_size, self.worker_refresh_batch_size
+                    self.num_workers_expected - num_workers_running, self.worker_refresh_batch_size
                 )
-                self.log.debug(
+                self.log.info(
```

Comment:
Any reason for changing the log level? @drago-f5a

Reply:
@kaxil 
This action (spawning new workers) is taken specifically to correct an issue that was detected and logged at the error level few lines of code above. Given the prior error level log, it seemed reasonable to log an attempt at corrective action at the info level.

Reply:
That makes sense, can you create a new PR please to change the log level back to info. I revert that changed before merging the PR

Reply:
@kaxil Sure, I'll do it and I'll tag you on it.